tools/ci_build/github/azure-pipelines/templates/win-ci-2019.yml

```diff
@@ -30,8 +30,6 @@ jobs:
     DotNetExe: 'dotnet.exe'
     CUDA_VERSION: ${{ parameters.CudaVersion }}
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
-    PlatformTarget: ${{ parameters.BuildArch }}
```

Comment:
These 2 lines are not needed and setting these was the reason the managed dll was built for x64.